### PR TITLE
GNOME updates 2024-06-25

### PR DESCRIPTION
**Pull request by JohnRTitor**
## Description of changes

`adwaita-icon-theme` not updated as it will be done as part of #316416.

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

File: pkgs/tools/networking/networkmanager/openvpn/default.nix [hunk 1]
```diff
@@ -21,11 +21,11 @@
 
 stdenv.mkDerivation rec {
   pname = "NetworkManager-openvpn";
-  version = "1.10.2";
```
Comment (jtojnar) [inline pkgs/tools/networking/networkmanager/openvpn/default.nix:24]: This requires NM 1.48.2. https://github.com/NixOS/nixpkgs/pull/321585#issuecomment-2183462987

Comment (jtojnar) [reply pkgs/tools/networking/networkmanager/openvpn/default.nix:24]: And while NetworkManager updates can go to master nowadays, stable is on NM 1.46 so it needs a separate update to 1.46.2.

Comment (JohnRTitor) [reply pkgs/tools/networking/networkmanager/openvpn/default.nix:24]: Does NM 1.46.2 supports networkmanager-openvpn 1.12.0?

We can do a partial backport of this to stable then.

**Comment by jtojnar** [comment]
The release notes mentioned 1.46.2 but I am not sure if it was not a typo.
Would need to double check that the relevant commit is included.

**Review by jtojnar — APPROVED** [review]
Looks like the commit is indeed in 1.46.2: https://gitlab.freedesktop.org/NetworkManager/NetworkManager/-/commit/dbae0113dbe2e2738a90d15abe038d40584a12f8

Changelogs look okay:

- [x] https://gitlab.freedesktop.org/NetworkManager/NetworkManager/-/compare/1.48.0...1.48.2
- [x] https://gitlab.gnome.org/GNOME/NetworkManager-openvpn/-/compare/1.10.2...1.12.0
- [x] https://gitlab.gnome.org/GNOME/gnome-sudoku/-/compare/46.1...46.2

**Comment by jtojnar** [comment]
Thanks.

**Pull request merged**